assets/scss/components/_quick-search.scss

```diff
@@ -3,38 +3,21 @@
    #QUICK-SEARCH
    ========================================================================== */
 
-.c-quick-search {
-    align-items: center;
-    background-color: $color-gray-e5;
-    display: flex;
-    justify-content: center;
-    margin-left: calc(50% - 50vw);
-    padding: 3rem;
-    width: 100vw;
+$quick-search-icon: './../icons/search.svg';
 
-    > div {
-        width: 100%;
-        max-width: 1024px;
-        margin-inline: auto;
-    }
+.c-quick-search .c-form__input {
```

Comment:
Is this nesting necessary here? Couldn't it be just `.c-quick-search`?

Reply:
.c-quick-search is a parent selector